instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -31,18 +32,23 @@ public final class LoggingEventMapper {
 
   private static final Cache<String, AttributeKey<String>> mdcAttributeKeys = Cache.bounded(100);
 
+  private static final AttributeKey<String> LOG_MARKER = AttributeKey.stringKey("log.marker");
```

Comment:
so that it doesn't look like a spec'd attribute:

```suggestion
  private static final AttributeKey<String> LOG_MARKER = AttributeKey.stringKey("logback.marker");
```

Reply:
Done